public/entry.worker.js

```diff
@@ -0,0 +1,278 @@
+// node_modules/@remix-run/router/dist/router.js
```

Comment:
mesma coisa aqui....to confiando que você sabe o que isso ta fazendo kkkkk

Reply:
well, I am trusting the remix-pwa lib.